util/notify/notify.asd

```diff
@@ -0,0 +1,13 @@
+;;;; notify.asd
+
+(asdf:compute-source-registry)
+(declaim (optimize (speed 0) (debug 3) (safety 3)))
```

Comment:
In a similar spirit than above, remove the declaim form. It is unspecified behaviour if declaim has any effect beyond the file being compiled. And if it has any effect (as I believe it persists in SBCL but I may be misremembering) it would affect other systems that are compiled afterwards. If you want to specify optimization settings for your system, which I do not think is a idea by default, add a declaim form at the start of each of the files in the system.